frontend/src/components/AboutUsSection/AboutUs.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Button } from '../shared/button';
+import ProgressBar from '../ProgressBar/ProgressBar';
+import { useEffect, useState } from 'react';
+import { API } from '../../modules/apis';
+
+const AboutUs = () => {
+  const [sectionData, setSectionData] = useState({});
```

Comment:
don't initialize the sate with an empty object.

just do useState();

this is what resulted in the unexpected behavior